### PR TITLE
Feat(BVM): change op div

### PR DESCRIPTION
**Pull request by songxuexian**

### CODE REVIEW

File: protocol/vm/numeric.go [hunk 1]
```diff
@@ -80,11 +80,22 @@ func op2Div(vm *virtualMachine) error {
...
 	}
-	n, err := vm.popInt64(true)
+
+	n, err := vm.popBigInt(true)
 	if err != nil {
 		return err
 	}
-	return vm.pushInt64(n>>1, true)
+
+	num, ok := checked.NewUInt256("2")
```
Comment (jacksoom) [inline protocol/vm/numeric.go:89]: Rsh(n, 2)

Comment (jacksoom) [reply protocol/vm/numeric.go:89]: func (z *Int) Rsh(x *Int, n uint) *Int {

Comment (jacksoom) [reply protocol/vm/numeric.go:89]: https://github.com/holiman/uint256/blob/3c4134ff99a6155c11159478278c3ab02e13d669/uint256.go#L892

File: protocol/vm/numeric.go [hunk 2]
```diff
@@ -80,11 +80,22 @@ func op2Div(vm *virtualMachine) error {
...
+	if !ok {
+		return ErrBadValue
+	}
+
+	if num.Div(n, num); num.Sign() < 0 {
```
Comment (jacksoom) [inline protocol/vm/numeric.go:94]: 貌似没有必要检查

File: protocol/vm/numeric.go (hunk 1 above)
Comment (jacksoom) [inline protocol/vm/numeric.go:89]: https://github.com/holiman/uint256/blob/3c4134ff99a6155c11159478278c3ab02e13d669/uint256.go#L892

File: protocol/vm/numeric.go (hunk 2 above)
Comment (songxuexian) [reply protocol/vm/numeric.go]: 💯

File: protocol/vm/numeric.go (hunk 1 above)
Comment (songxuexian) [reply protocol/vm/numeric.go]: 🥇

**Review by jacksoom — APPROVED** [review]

File: protocol/vm/numeric_test.go [hunk 3]
```diff
@@ -928,3 +928,75 @@ func Test_op1Sub(t *testing.T) {
...
+				vm: &virtualMachine{
+					runLimit:  50000,
+					dataStack: [][]byte{{0x02}},
+				},
+			},
+			want:    [][]byte{{0x01}},
+			wantErr: false,
+		},
+		{
+			name: "Test that two bytes number become one byte number after op sub",
```
Comment (jacksoom) [inline protocol/vm/numeric_test.go:955]: 注释？

Comment (songxuexian) [reply protocol/vm/numeric_test.go]: 👀

**Review by Paladz — APPROVED** [review]

**Pull request merged**